src/pages/home/report/ReportActionItem.js

```diff
@@ -360,6 +371,7 @@ class ReportActionItem extends Component {
                                 isVisible={hovered && !this.props.draftMessage && !hasErrors}
                                 draftMessage={this.props.draftMessage}
                                 isChronosReport={ReportUtils.chatIncludesChronos(this.props.report)}
+                                childReportActionID={this.props.action.childReportActionID}
```

Comment:
I don't see why we are passing `childReportActionID` here.

Reply:
cc @stitesExpensify

Reply:
This is a relic of when I had the button logic doing different things, but instead we're doing it in navigateTo... 

We can remove this I think